blocks/sticky-footer/sticky-footer.css

```diff
@@ -81,6 +81,10 @@
   margin-bottom: var(--space-3px);
 }
 
+.block.sticky-footer .action-container input::placeholder {
+  color: var(--color-strong-gray-light);
```

Comment:
Nit: seems the current site text color is lighter than EDS page's
<img width="282" alt="image" src="https://github.com/aemsites/icicidirect/assets/57093988/acb65392-22cb-4d29-b6aa-63056fe1207c">

Reply:
Also please check once the contrast ratio for the placeholder text. If we keep it light grey, it would affect the accessibility score.

Reply:
addressed, now check the accessibility score.

Reply:
this section is not there in the contrasts related issues, so we are good to merge this